### DISCUSSION TITLE
Deploy Razzle to Plesk ?

### DISCUSSION THREAD

**Discussion by ghost** [jaredpalmer/razzle#1517] [Q&A] [answered]
I am struggling with the deployment process of Razzle. In react it was only` yarn build `and zip the files upload it to the server extract there and that's it. It will work but on Razzle it is different I dont see index.html after running `yarn build`.?

**Comment by fivethreeo** [comment, marked as answer]
https://razzle-git-canary.jared.vercel.app/deployment-options/plesk

**Reply by ghost** [reply]
@fivethreeo glad to see there is a guide on how to deploy to Plesk
thank you
